### PR TITLE
Adding Windows Server and Container build and publish workflows for Concourse

### PR DESCRIPTION
**Pull request by bkatyl**
Adding Windows Server and Container build and publish workflows for Concourse

### CODE REVIEW

File: daisy_workflows/build-publish/windows_container/windows-2019-for-containers-uefi.publish [hunk 1]
```diff
@@ -0,0 +1,57 @@
+{{/*
```
Comment (hopkiw) [inline daisy_workflows/build-publish/windows_container/windows-2019-for-containers-uefi.publish:1]: i think this filename is incorrect

Comment (bkatyl) [reply daisy_workflows/build-publish/windows_container/windows-2019-for-containers-uefi.publish]: Thanks for catching that. Updating filename to be .publish.json.

File: daisy_workflows/build-publish/windows/windows-server-2004-dc-core-uefi.publish.json [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+  {{$work_project := printf "%q" "gce-image-builder" -}}
+  {{$endpoint := `"https://www.googleapis.com/compute/alpha/projects/"` -}}
+  {{$delete_after := `"24h*30*6"` -}}
+  {{if eq .environment "prod" -}}
+  "WorkProject": {{$work_project}},
+  "PublishProject": "windows-cloud",
+  "ComputeEndpoint": {{$endpoint}},
+  {{- else if eq .environment "staging" -}}
+  "WorkProject": "bct-staging-images",
+  "PublishProject": "bct-staging-images",
```
Comment (zmarano) [inline daisy_workflows/build-publish/windows/windows-server-2004-dc-core-uefi.publish.json:21]: For staging change everything to use gce-staging-images instead (we're migrating to that project)

**Review by zmarano — APPROVED** [review]

File: daisy_workflows/build-publish/windows/windows-server-2004-dc-core-uefi.publish.json [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+  Template to publish Windows server UEFI images.
+  By default this template is setup to publish to the 'gce-image-builder'
+  project, the 'environment' variable can be used to publish to 'test', 'prod',
+  or 'staging'.
+  DeleteAfter is set to 180 days for all environments other than prod where no
+  time period is set.
+*/}}
+{
+  "Name": "windows-server-2004-dc-core-uefi",
+  "SourceGCSPath": "gs://gce-image-archive/windows-uefi",
```
[the remark this answers is not in the document]
Comment (bkatyl) [reply daisy_workflows/build-publish/windows/windows-server-2004-dc-core-uefi.publish.json]: Done

File: daisy_workflows/build-publish/windows/windows-server-2004-dc-core-uefi.publish.json (hunk 2 above)
Comment (bkatyl) [reply daisy_workflows/build-publish/windows/windows-server-2004-dc-core-uefi.publish.json]: Done

**Review by zmarano — APPROVED** [review]

**Pull request merged**